command/command.go

```diff
@@ -229,3 +238,34 @@ func computeNewlineTargets(before, after string) []int {
 
 	return result
 }
+
+func preprocessYAML(content []byte) ([]byte, error) {
+	var cleaned []string
+	lines := strings.Split(string(content), "\n")
+
+	for _, line := range lines {
+		cleanedLine := strings.TrimRight(line, " \t")
+		cleaned = append(cleaned, cleanedLine)
+	}
+
+	return []byte(strings.Join(cleaned, "\n")), nil
+}
+
+func processMultilineNodes(node *yaml.Node) {
+	if node == nil {
+		return
+	}
+
+	// Check if the node is a scalar and contains newlines
+	if node.Kind == yaml.ScalarNode && strings.Contains(node.Value, "\n") {
```

Comment:
What happens if there is a newline escaped in the node?
```
run: |
  echo "Hello\n"
  echo "World\n"
```

Reply:
As far as I can see by running it, nothing unexpected 😄 

Input Snippet:
```yaml
  steps:
    - name: Execute Tests
      run: |
        echo "Starting tests..."      
        echo "Hello\n"
        echo "World\n"
        
        echo "Running test suite..."  
        npm test        

        echo "Tests complete."      
```

Output Snippet:
```yaml
  steps:
    - name: Execute Tests
      run: |
        echo "Starting tests..."
        echo "Hello\n"
        echo "World\n"

        echo "Running test suite..."
        npm test

        echo "Tests complete."
```